inc/user.class.php

```diff
@@ -107,6 +107,17 @@ static function getTechnicianGroup($entity, $userid, $first = true) {
       return self::getUserGroup($entity, $userid, '`is_assign`', $first);
    }
 
+   static function getUserDefaultGroup($users_id) {
+      $groups_id = 0;
+
+      $user = new User();
+      $user->getFromDB($users_id);
+      if ($user->getField('groups_id')) {
+         $groups_id = $user->getField('groups_id');
+      }
+      return $groups_id;
```

Comment:
`getField()` will return `N/A` if value is not a valid group id. This method should only be used in display context.

```suggestion
      $groups_id = 0;

      $user = new User();
      if ($user->getFromDB($users_id) && $user->fields['groups_id']) {
         $groups_id = $user->fields['groups_id'];
      }
      return $groups_id;
```

Reply:
thank you, I've used $user->fields instead of $user->getField